docs/en/sql-reference/statements/create/dictionary.md

```diff
@@ -18,6 +18,7 @@ CREATE [OR REPLACE] DICTIONARY [IF NOT EXISTS] [db.]dictionary_name [ON CLUSTER
     attr2 type2 [DEFAULT|EXPRESSION expr4] [HIERARCHICAL|INJECTIVE]
 )
 PRIMARY KEY key1, key2
+SHARDS(10) /* for HASHED/SPARSE_HASHED only */
```

Comment:
Need to make this only part of hash dictionary layout configuration.

Reply:
I will move it to layout parameters, like `PREALLOCATE`, though it is not checked either.